package.json

```diff
@@ -343,8 +345,7 @@
         "trailingComma": "es5",
         "bracketSpacing": true,
         "arrowParens": "avoid",
-        "endOfLine": "lf",
-        "singleQuote": true
```

Comment:
I hope this doesn't cause new eslint issues

Reply:
Will go away after I have synced my repo.